packages/gui/src/electron/utils/openReactDialog.tsx

```diff
@@ -0,0 +1,199 @@
+  const initialBgColor = isDarkMode ? '#0f252a' : '#ffffff';
+
+  try {
+    const scriptAssetURL = addAsset({
+      content: generateScriptContent(confirmId),
+      type: 'text/javascript',
+    });
+
+    const cssPath = path.join(__dirname, 'main.css');
+    const cssContent = fs.readFileSync(cssPath, 'utf-8');
+
+    const styleAssetURL = addAsset({
+      content: cssContent,
+      type: 'text/css',
+    });
+
+    const html = renderToStaticMarkup(
+      <html lang="en" className={isDarkMode ? 'dark' : ''}>
+        <head>
+          <meta charSet="utf-8" />
+          <meta
+            httpEquiv="Content-Security-Policy"
+            content="
+              default-src 'none';
+              base-uri   'none';
+              connect-src 'none';
+              font-src   'none';
+              frame-src  'none';
+              img-src data:;
+              media-src 'none';
+              object-src 'none';
+              script-src asset:;
+              style-src  asset:;
+              worker-src 'none';
+              form-action 'none';
+            "
+          />
+          <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes" />
+          <link href={styleAssetURL} type="text/css" rel="stylesheet" />
+        </head>
+        <body>
+          <Component {...props} confirmId={confirmId} styleURL={styleAssetURL} isDarkMode={isDarkMode} />
+          <script src={scriptAssetURL} />
+        </body>
+      </html>,
+    );
+
+    const dialogURL = addAsset({
+      content: `<!DOCTYPE html>${html}`,
+      type: 'text/html',
+    });
+
+    const assets = [dialogURL, scriptAssetURL, styleAssetURL];
+
+    let settled = false;
+
+    return new Promise<TResponse | undefined>((resolve, reject) => {
+      function safeResolve(value: TResponse | undefined) {
+        if (settled) {
+          return;
+        }
+        settled = true;
+        resolve(value);
+      }
+      function safeReject(error: Error) {
+        if (settled) {
+          return;
+        }
+        settled = true;
+        reject(error);
+      }
+
+      const dialog = new BrowserWindow({
+        modal,
+        parent,
+        width,
+        height,
+        title,
+        show: false,
+        resizable: false,
+        useContentSize: true,
+        titleBarStyle,
+        backgroundColor: initialBgColor,
+        webPreferences: {
+          preload: path.join(__dirname, 'preloadDialog.js'),
+          contextIsolation: true,
+          sandbox: true,
+          nodeIntegration: false,
+          additionalArguments: [`--resolveChannelId=${resolveChannelId}`, `--rejectChannelId=${rejectChannelId}`],
+        },
+      });
+
+      dialog.webContents.on('will-navigate', (event, url) => {
+        event.preventDefault();
+        const currentURL = dialog.webContents.getURL();
+
+        if (url !== currentURL) {
+          event.preventDefault();
+          openExternal(url);
```

Comment:
Under what circumstances this path is handled?

Reply:
if you are in about dialog and click on report issue, or click on logo it will be opened in your browser